### PR TITLE
feat(client): add support for `mermaid-js`

### LINKED ISSUE #502: mermaid-js integration
automatically having access to https://github.com/mermaid-js/mermaid within all docs would be pretty cool!

The goal is being able to write the following within the documents:

``````
```mermaid

graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;

```
``````

### PR DESCRIPTION
**Pull request by charlypoly**
Closes #502

### CODE REVIEW

**Comment by charlypoly** [comment]
@B2o5T I'm not sure which packages (besides the `client`) should I bump, could you help me on this? 🙏🏼

**Review by saihaj — APPROVED** [review]

File: .changeset/flat-bikes-attack.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"guild-docs": patch
+"@guild-docs/client": patch
```
Comment (saihaj) [inline .changeset/flat-bikes-attack.md:3]: This is a new feature : ) 
```suggestion
"@guild-docs/client": minor
```

File: .changeset/flat-bikes-attack.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"guild-docs": patch
```
Comment (B2o5T) [inline .changeset/flat-bikes-attack.md]: guild-docs is a cli package, you didn't changed it

**Comment by B2o5T** [comment]
> @B2o5T I'm not sure which packages (besides the client) should I bump, could you help me on this? 🙏🏼
@charlypoly I think only `@guild-docs/client`

**Review by B2o5T — APPROVED** [review]
ty @charlypoly 😍

**Pull request merged**